app-modules/ai/src/Services/Contracts/AiService.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace AdvisingApp\Ai\Services\Contracts;
+
+use AdvisingApp\Ai\Models\AiAssistant;
+use AdvisingApp\Ai\Models\AiThread;
+use AdvisingApp\Ai\Models\AiMessage;
+
+interface AiService
+{
+    /**
+     * This method is passed an unsaved `AiAssistant` model and should return
+     * the model with any additional data that associates it with
+     * the AI service, such as the `assistant_id`.
+     */
+    public function createAssistant(AiAssistant $assistant): void;
```

Comment:
Based on the the description of what this does here shouldn't the return type here be `AiAssistant` not `void`?

Reply:
It modifies the existing object so it makes no difference, just saves lines not having to return